.devcontainer/devcontainer.json

```diff
@@ -14,9 +14,11 @@
   },
 
   // Use 'forwardPorts' to make a list of ports inside the container available locally.
-  "forwardPorts": [3000, 8000, 8080]
+  "forwardPorts": [3000, 8000, 8080],
 
   // Use 'postCreateCommand' to run commands after the container is created.
   // Note: Useful for when they open it and want everything to just start
   // "postCreateCommand": "docker compose up -d && yarn dev"
+
+  "onCreateCommand": "git config --global --add safe.directory /workspace"
```

Comment:
This path doesn't exist. Maybe change to . but I believe it's unnecessary

Reply:
i think its necessary because i've seen the unsafe directory sometimes